src/Agent.Worker/JobRunner.cs

```diff
@@ -41,6 +41,87 @@ public sealed class JobRunner : AgentService, IJobRunner
 
             DateTime jobStartTimeUtc = DateTime.UtcNow;
 
+            var tfsGit1 = new Pipelines.RepositoryResource()
```

Comment:
?

Reply:
this is for me to test my change, since the server haven't sent down repository as part of job message.